frontend/templates/product/sections/ReviewsSection/index.tsx

```diff
@@ -220,7 +243,30 @@ type RatingCount = {
 };
 
 function formatReviewDate(timeSeconds: number) {
-   return new Intl.DateTimeFormat('en-US', { dateStyle: 'long' }).format(
-      new Date(timeSeconds * 1000)
-   );
+   const formatter = new Intl.RelativeTimeFormat('en-US', { numeric: 'auto' });
+   const date = new Date(timeSeconds * 1000);
+   let duration = (date.getTime() - new Date().getTime()) / 1000;
+
+   for (let i = 0; i <= DIVISIONS.length; i++) {
+      const division = DIVISIONS[i];
```

Comment:
Not worth blocking on, but I think we prefer `for (const division of DIVISIONS) { ... }` for readability if it's possible here